ds-css-v2/styles/sass/cbp-theme-settings/_cbp-header.scss

```diff
@@ -244,7 +244,7 @@ $mobile-breakpoint: uswds-mobile-breakpoint.$theme-header-min-width - 1px;
 
   ul {
     padding: 0;
-    margin-top: $cbp-size-1;
+    margin-top: 8px; // $cbp-size-1;
```

Comment:
@dannyk3941 Is there no pixel token for 8px?

Reply:
Resolved. The sizing tokens will be replaced once they are determined by the team.